.github/workflows/main.yml

```diff
@@ -26,23 +25,20 @@ jobs:
     strategy:
       matrix:
         python-version: [3.5, 3.6, 3.7, 3.8]
-
     # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
       - uses: actions/checkout@v2
-      - name: Set up Python ${{ matrix.python-version }}
-        uses: actions/setup-python@v2
-        with:
-          python-version: ${{ matrix.python-version }}
-      - name: Install dependencies
-        run: |
-          python -m pip install --upgrade pip
-          pip install -r requirements.txt -r dev-requirements.txt
-
-      # Run flake8
-      - name: Run flake8
-        run: flake8 --ignore=E501,E722,W504,W503
-      # Runs tests
       - name: Run tests
-        run: FLASK_ENV=testing pytest --doctest-modules -n 4 --dist=loadfile -v OpenOversight/tests/ OpenOversight/app;
+        run: PYTHON_VERSION=${{ matrix.python-version }} make test_with_version
+  pre-commit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+      - name: Install Dependencies
+        run: pip install pre-commit
+      - name: Run pre-commits
+        run: pre-commit run --all-files
```

Comment:
There is a [standalone pre-commit Github action](https://github.com/pre-commit/action#using-this-action) we could use which may be simpler

Reply:
Nice. Changed this.